addons/default/visiosoft/advs-module/resources/views/ad-detail/partials/content.twig

```diff
@@ -26,7 +26,7 @@
                 </div>
 
                 <!-- Content With Block -->
-                {{ addBlock('ad-detail/content',{'features':features,'adv':adv})|raw }}
+                {{ addBlock('ad-detail/content',{'features':features,'adv':adv, 'qrSRC': qrSRC})|raw }}
```

Comment:
do we need to qrSRC?

Reply:
yes for qr contact module

Reply:
we can just show qrcode by sending ad and profile info in twig. Otherwise every module will add some data to core project. it will mess @Diatrex